legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/extension/PureGrammarParserExtension.java

```diff
@@ -61,4 +67,26 @@ default Iterable<? extends MappingIncludeParser> getExtraMappingIncludeParsers()
     {
         return Collections.emptyList();
     }
+
+    static String parseIncludedStoreType(String type, List<Function<String, String>> processors)
```

Comment:
Maybe move this out to the MappingParserHelper or something?

Reply:
Makes sense. Done